modules/module-info.php

```diff
@@ -272,14 +210,9 @@ function jetpack_subscriptions_load_more_link() {
 add_action( 'jetpack_learn_more_button_subscriptions', 'jetpack_subscriptions_load_more_link' );
 
 function jetpack_subscriptions_more_info() { ?>
-	<div class="jp-info-img">
-		<img class="jp-info-img" src="<?php echo plugins_url( basename( dirname( dirname( __FILE__ ) ) ) . '/images/screenshots/subscriptions.jpg' ) ?>" alt="<?php esc_attr_e( 'Subsriptions Screenshot', 'jetpack' ) ?>" width="300" height="150" />
-	</div>
-
 	<?php esc_html_e(
-		'Allow any visitor to subscribe to your site through a widget in your sidebar. Each time you publish new content,
-		Jetpack will email a notification to all of your subscribers. When leaving comments, your visitors can also
-		subscribe to a post’s comments to keep up with the conversation.'
+		'A widget in your sidebar allows visitors to subscribe to your site so that they receive an email
+		each time you publish new content. Your visitors can also	subscribe to a post’s comments to keep up with the conversation.'
```

Comment:
Ah also, `subscribe to post’s comments` is using the `’` character instead of `\'` as in other places.